### PR TITLE
fix: add edited and server hints to author name in quotes

### PR DESCRIPTION
**Pull request by DorraJaouad**
### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [x] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/composables/useMessageInfo.js [hunk 1]
```diff
@@ -67,6 +70,36 @@ export function useMessageInfo(message = ref({})) {
...
+	function getRemoteServer() {
+		return message.value.actorType === ATTENDEE.ACTOR_TYPE.FEDERATED_USERS
+			? '(' + message.value.actorId.split('@').pop() + ')'
+			: ''
+	}
+
+	/**
+	 *
+	 */
+	function getLastEditor() {
```
Comment (Antreesy) [inline src/composables/useMessageInfo.js:85]: Maybe convert them to computed props? Should be more optimal (at least in Vue3)

File: src/components/Quote.vue [hunk 2]
```diff
@@ -25,6 +25,8 @@ components.
 					:size="AVATAR.SIZE.EXTRA_SMALL"
 					disable-menu />
 				{{ getDisplayName }}
+				<span v-if="isFederatedUser" class="messages__author-server">{{ getRemoteServer() }}</span>
+				<span v-if="message.lastEditTimestamp" class="messages__author-edit">{{ getLastEditor() }}</span>
```
Comment (Antreesy) [inline src/components/Quote.vue:29]:
```suggestion
				<span v-if="isFederatedUser" class="quote__main__author-server">{{ getRemoteServer() }}</span>
				<span v-if="message.lastEditTimestamp" class="quote__main__author-edit">{{ getLastEditor() }}</span>
```

File: src/components/Quote.vue [hunk 3]
```diff
@@ -279,6 +289,13 @@ export default {
 			align-items: center;
 			gap: 4px;
 			color: var(--color-text-maxcontrast);
+
+			&-edit,
+			&-server {
+				white-space: nowrap;
+				overflow: hidden;
+				text-overflow: ellipsis;
```
Comment (Antreesy) [inline src/components/Quote.vue:297]: still, not ellipsized 
image

add `width: 100%` to quote__main

**Comment by nickvergessen** [comment]
> - ⛑️ Tests are included

Please also make this if possible :)

File: src/components/MessagesList/MessagesGroup/MessagesGroup.vue [hunk 4]
```diff
@@ -75,10 +78,18 @@ export default {
 		},
 	},
 
-	setup() {
+	setup(props) {
+		const { messages } = toRefs(props)
+		const firstMessage = computed(() => messages.value[0])
```
[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/MessagesList/MessagesGroup/MessagesGroup.vue:83]: I don't see a reason to do so? can be done swiftly when migrating to composition API

**Review by Antreesy — APPROVED** [review]
Nice

**Pull request merged**